client/blocks/login/social.scss

```diff
@@ -42,7 +42,7 @@
 						border: 1px solid var(--studio-gray-50, #646970);
 						border-radius: 4px;
 						display: flex;
-						height: 40px;
+						height: 44px;
```

Comment:
50px looks a lot better but I opted for 44 to match the height of the continue button on the left. This also prevents the forgot password link falling below the fold on some smaller mobile screens.